pkgs/development/python-modules/consul2/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  requests,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "python-consul2";
+  version = "0.1.5";
+
+  src = fetchPypi {
```

Comment:
Use `fetchFromGitHub` if possible, since it would make inspecting the upstream source code easier and make sure we are really building from "source".